sceptre/strides-ampad-workflows/config/prod/lambda-finops-emails.yaml

```diff
@@ -0,0 +1,9 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/lambda-finops-email-strides/1.0.0/lambda-finops-s3-cost-report.yaml
+stack_name: lambda-finops-s3-cost-report
```

Comment:
as a convention we usually give the stack and file the same name.

Reply:
oh whoops, I updated the typo in the url, but not the filename